packages/next/server/lib/find-page-file.ts

```diff
@@ -17,6 +16,15 @@ async function isTrueCasePagePath(pagePath: string, pagesDir: string) {
   return (await Promise.all(segmentExistsPromises)).every(Boolean)
 }
 
+async function isReadable(directory: string): Promise<boolean> {
```

Comment:
You can use the `fileExists` function from `packages/next/lib/file-exists.ts` instead which does the same only checking for `ENOENT` to ensure filesystem errors are throwing:

```
await fileExists(directory)
```

Reply:
Done!